Cargo.toml

```diff
@@ -12,19 +12,22 @@ repository = "https://github.com/fluffysquirrels/mqtt-async-client-rs"
 bytes = "0.4.0"
 futures-core = "0.3.1"
 futures-util = "0.3.1"
+http = "0.2.4"
```

Comment:
Can `http` be optional and included on the "websocket" feature?

Reply:
Good catch